### DISCUSSION TITLE
How do I delete a file that I forgot to save the edit link of?

### DISCUSSION THREAD

**Discussion by SinTan1729** [epoupon/fileshelter#81] [General]
I created a local  share using `fileshelter-create` and forgot to save the edit link anywhere. How can I delete the share?

**Comment by SinTan1729** [comment]
Also, is it possible to disable the TOS page?

**Reply by epoupon** [reply]
Not from the config file, but this can be done.

**Reply by SinTan1729** [reply]
Is it doable without possibly rebuilding the image?

**Reply by epoupon** [reply]
Unfotunately not. You can just use a custom, empty tos, but there will be a tos entry in the top menu.

**Comment by epoupon** [comment]
> I created a local share using `fileshelter-create` and forgot to save the edit link anywhere. How can I delete the share?

I planned to add the edit URL in ´fileshelter-list´, see  #79
